scout/parse/variant/genotype.py

```diff
@@ -264,6 +346,13 @@ def get_alt_depth(
             if split_read_alt:
                 alt_depth += split_read_alt
 
+        if any([clip5_alt, clip3_alt]):
```

Comment:
These are in line with the other SV "support" sums: a fairly advanced mathematical model would be needed to give a proper guesstimate of zygosity, but the counts of soft clips ajacent to insertion does give a hint.